src/javascript/campaign-url-builder/params.js

```diff
@@ -14,6 +14,10 @@
 
 
 import qs from 'querystring';
+
+// TODO(nathanwest): url isn't actually a dependency of this project.
+// it seems like url-parse is the better library, anyway, so switch
+// to that at some point.
 import url from 'url';
```

Comment:
Sure, `parse-url` seems fine to me.

You could also use [this module](https://github.com/philipwalton/dom-utils/blob/master/lib/parse-url.js), which is already being loaded since it's used in autotrack, and it's smaller and has pretty much the same API.

```js
import parseUrl from 'dom-utils/lib/parse-url';
```